distros/openshift/security-context-constraints/README.md

```diff
@@ -0,0 +1,9 @@
+# distros/openshift/security-context-constraints
+
+## Security Context Constraints
+
+OpenShift deployments require Security Context Constraints to be applied for containers to function properly. The ones defined here were identified through testing the deployment against an OpenShift cluster and creating SCCs. These are prerequisites, and should be applied before attempting to install Nephio components.
+
+## Storage
+
+Gitea requires storage to be configured on the OpenShift cluster. Whether LVM, Local Storage, etc. there needs to be storage and a default Storage Class for the Persistent Volumes to be allocated.
```

Comment:
Is this part of security constraints or should it be mentioned in the overall guide somewhere? With examples on how to?

Reply:
Yes, I think checking the docs for this content and making sure they reflect this in a follow up PR would be good

Reply:
https://github.com/nephio-project/nephio/issues/801